FreeAgent.postman_collection.json

```diff
@@ -1159,8 +1150,161 @@
 					},
 					"response": []
 				}
-			],
-			"protocolProfileBehavior": {}
+			]
+		},
+		{
+			"name": "Bill Items",
+			"item": [
+				{
+					"name": "Index",
+					"request": {
+						"auth": {
+							"type": "oauth2",
+							"oauth2": [
+								{
+									"key": "useBrowser",
+									"value": false,
+									"type": "boolean"
+								},
+								{
+									"key": "client_authentication",
+									"value": "header",
+									"type": "string"
+								},
+								{
+									"key": "clientSecret",
+									"value": "{{client_secret}}",
+									"type": "string"
+								},
+								{
+									"key": "clientId",
+									"value": "{{client_id}}",
+									"type": "string"
+								},
+								{
+									"key": "accessTokenUrl",
+									"value": "{{url}}/v2/token_endpoint",
+									"type": "string"
+								},
+								{
+									"key": "authUrl",
+									"value": "{{url}}/v2/approve_app",
+									"type": "string"
+								},
+								{
+									"key": "redirect_uri",
+									"value": "https://www.getpostman.com/oauth2/callback",
+									"type": "string"
+								},
+								{
+									"key": "tokenName",
+									"value": "token",
+									"type": "string"
+								},
+								{
+									"key": "addTokenTo",
+									"value": "header",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{url}}/v2/bill_items?bill=https://api.int13.fa-app-int.com/v2/bills/3",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"v2",
+								"bill_items"
+							],
+							"query": [
+								{
+									"key": "bill",
+									"value": "https://api.int13.fa-app-int.com/v2/bills/3"
```

Comment:
```suggestion
									"value": "{{url}}/v2/bills/3"
```

Reply:
It would be worth replacing the environment-specific URLs with more generic ones.